doc/mgr/ansible.rst

```diff
@@ -120,3 +120,56 @@ Example::
     sda (hdd, 53687091200b)
     sdb (hdd, 53687091200b)
     sdc (hdd, 53687091200b)
+
+**Create OSD:**
+
+Create an OSD in one or more hosts using a drive group specification:
+
+orchestrator osd create {<osd_host>} {<json>} {<file>}
+
+Where:
+
+{<osd_host>}: Optional parameter to create OSD only in this host
+{<json>}: String with json format used to specify 'lvm_volumes' parameter
+{<file>}: Name of the file that contains a json string to specify 'lvm_volumes' parameter
+
+The "lvm_volumes" parameter is a list of dictionaries that contains the Drive group attributes
+
+A Drive Group: Is set of parameters used to specify how to use the storage devices in one host in order to create one or more OSD's.
+
+Not needed to specify all the parameters, if one parameter does not appear, the value for this parameter is obtained from the current Ceph Ansible cluster configuration.
+(parameter found in "all.yml" or "osd.yml" files in group vars folder)
+
+{
+"objectstore": filestore/bluestore,
+"dmcrypt": true/false,
+"osds_per_device": <number>
+"data": List of data devices,
+"shared_devices": List of devices used for db/journal
+"fast_shared_devices": List of devices used for wal
+"Number of slots": #how many  data devices per shared device, independently of len(data_devices)
+"Underprovision_ratio": # leave room on shared device
+"Number of fast slots":
+"underprovision_fast_ratio":
+"extended_attributes": {} #A dict with future extended attributes
+ }
+
+Example:
+[root@mon0 ~]# ceph orchestrator osd create --osd_host 192.168.121.142 --json "{'lvm_volumes':[{'data': '/dev/sdc'}]}"
+
+Create an OSD using the device '/dev/sdc' present in host '192.168.121.142', using the rest of parameters used by default for this cluster.
+
+
+
+**Remove OSD:**
```

Comment:
could you just reference the common documentation and just document things special to Ansible ?

Reply:
Agree